chapter_appendix-mathematics-for-deep-learning/geometry-linear-algebraic-ops.md

```diff
@@ -60,13 +60,13 @@ as discovering how to separate two distinct clusters of points.
 
 In parallel, there is a second point of view 
 that people often take of vectors: as directions in space. 
-Not only can we think of the vector $\mathbf{v} = [2,3]^\top$ 
-as the location $2$ units to the right and $3$ units up from the origin,
+Not only can we think of the vector $\mathbf{v} = [3,2]^\top$ 
+as the location $3$ units to the right and $2$ units up from the origin,
 we can also think of it as the direction itself 
-to take $2$ steps to the right and $3$ steps up. 
+to take $3$ steps to the right and $2$ steps up. 
 In this way, we consider all the vectors in figure :numref:`fig_arrow` the same.
 
-![Any vector can be visualized as an arrow in the plane.  In this case, every vector drawn is a representation of the vector $(2,3)$.](../img/ParVec.svg)
+![Any vector can be visualized as an arrow in the plane.  In this case, every vector drawn is a representation of the vector $(3,2)$.](../img/ParVec.svg)
```

Comment:
Could you change it to "[3,2]^\top"? Thanks!

Reply:
Ok!

Reply:
47a0df3ec7b1d08a2567155328a885c413ee2dea
The (2,3) in fig. 18.1.2 should be (3,2), but I don't know how to modify the figure. Do you know?

Reply:
No worries. I will handle the plot.

Reply:
Please let me know when you are done with all the modifications. Thanks for your contributions! @netyster :)

Reply:
@goldmermaid All the modification is done.:)